### PR TITLE
Bumps cypress from 5.2.0 to 6.6.0

### PR DESCRIPTION
**Pull request by conbrad**
Based on: https://docs.cypress.io/guides/references/migration-guide.html#Migrating-cy-route-to-cy-intercept

### CODE REVIEW

**Review by Kyubinhan — COMMENTED** [review]
It seems that formatting hasn't been run. Can you install Prettier plugin in your IDE and run it against all the modified files?

**Comment by conbrad** [comment]
> It seems that formatting hasn't been run. Can you install Prettier plugin in your IDE and run it against all the modified files?

Done

**Review by Kyubinhan — APPROVED** [review]

File: web/package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
     "@storybook/node-logger": "^6.1.11",
     "@storybook/preset-create-react-app": "^3.1.5",
     "@types/jest": "^24.9.1",
-    "cypress": "^5.2.0",
+    "cypress": "6.6.0",
```
Comment (Sybrand) [inline web/package.json:87]: Is there a reason we want to lock the minor?
```suggestion
    "cypress": "^6.6.0",
```

**Review by Sybrand — APPROVED** [review]
Looks pretty straight forward. Just a question/suggestion re. locking the version.

File: web/package.json (hunk 1 above)
Comment (conbrad) [reply web/package.json:87]: Good question. I copied the specific update command from somewhere on the cypress site but don't remember where. I'll update to not lock the minor.

**Pull request merged**